### PR TITLE
Update JUnit5 M1

### PR DESCRIPTION
**Pull request by pocketberserker**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -31,11 +34,21 @@ repositories {
...
+    testCompile "org.junit.jupiter:junit-jupiter-api:${junitJupiterVersion}"
+    testRuntime "org.junit.jupiter:junit-jupiter-engine:${junitJupiterVersion}"
+    testRuntime "org.apache.logging.log4j:log4j-core:${log4JVersion}"
+    testRuntime "org.apache.logging.log4j:log4j-jul:${log4JVersion}"
 }
 
-junit5 {
-    version '5.0.0-ALPHA'
+junitPlatform {
+    logManager 'org.apache.logging.log4j.jul.LogManager'
```
Comment (PoohSunny) [inline build.gradle:44]: Don't we need `platformVersion` here?  it is described in [junit5 user guide](http://junit.org/junit5/docs/current/user-guide/#running-tests-build) .

Comment (pocketberserker) [inline build.gradle:44]: `platformVersion` already set default value(https://github.com/junit-team/junit5/blob/r5.0.0-M1/junit-platform-gradle-plugin/src/main/groovy/org/junit/platform/gradle/plugin/JUnitPlatformExtension.groovy#L24).

**Comment by PoohSunny** [comment]
Understood. And I learned [JUnit5 gradle sample](https://github.com/junit-team/junit5-samples/blob/master/junit5-gradle-consumer/build.gradle#L37) does not have `platformVersion` in `junitPlatform` block. Then We don't need `platformVersion` here.

File: build.gradle (hunk 1 above)
Comment (PoohSunny) [inline build.gradle:44]: Understood. And I learned [JUnit5 gradle sample](https://github.com/junit-team/junit5-samples/blob/master/junit5-gradle-consumer/build.gradle#L37) does not have `platformVersion` in `junitPlatform` block. Then We don't need `platformVersion` here.

**Comment by PoohSunny** [comment]
👍

**Pull request merged**